build/detect-secrets-hook.js

```diff
@@ -27,7 +26,6 @@ function detectSecretsHook(reporter) {
 				break;
 		}
 		reporter(message, true);
-		throw new Error(message);
```

Comment:
Error doesn't need to be forcefully thrown anymore, because we ensure the hook fails in `hygiene.js`